File: xtend-gen/net/aveyon/intermediate_solidity/SmartContractModel.java
package net.aveyon.intermediate_solidity;

import java.util.LinkedList;
import java.util.List;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@SuppressWarnings("all")
public abstract class SmartContractModel {
  private String pragma;
  
  private List<SmartContract> smartContracts = new LinkedList<SmartContract>();
}

File: src/net/aveyon/intermediate_solidity/SmartContractModel.kt
package net.aveyon.intermediate_solidity

import java.util.LinkedList

abstract class SmartContractModel(
    val definitions: SolidityConcepts,
    val pragma: String,
    val license: String
) : Node("") {
    val imports: MutableList<ImportedConcept> = LinkedList()
}
